src/app/pages/voices/voices.component.scss

```
@@ -10,14 +10,14 @@ body {
 }
 
 h2 {
-    font-family: Manrope;
+    font-family: var(--sg-font-heading);
     font-weight: 700;
     font-style: Bold;
```

Stylelint (17.14.0)

[error] Expected "Bold" to be "bold" (value-keyword-case)

(value-keyword-case)